src/app/public/src/modules/blockquote/blockquote.component.scss

```diff
@@ -0,0 +1,27 @@
+@import '../../styles/_variables';
+
+.stache-blockquote {
+  border-left: 4px solid $stache-background-color-primary-dark;
+  font-size: $stache-font-size-base * 1.12;
+  color: $stache-text-color-deemphasized;
+  padding: $stache-padding * 3;
```

Comment:
This PR is perfect except for the padding. The padding-left doesn't match up with the table of contents. 

Change this to `15px 20px` and merge!